Packs/CortexXpanse/Wizards/wizard-Xpanse_ASM.json

```diff
@@ -0,0 +1,126 @@
+{
+    "id": "Xpanse Use Case Adoption Wizard",
+    "version": -1,
+    "modified": "2022-04-10T11:55:54.250933+03:00",
+    "fromVersion": "6.9.0",
+    "name": "Xpanse Use Case Adoption Wizard",
+    "description": "Welcome to the Deployment Wizard!  These steps will guide you through configuring your content pack so you'll have a working use case when the wizard finishes. Make sure to follow the steps in order, you can always continue the wizard where you left off. Learn more about this pack’s deployment in the \"Cortex XSOAR Administrator Guide\" -> \"Set up Your Use Case with the Deployment Wizard\".",
+    "dependency_packs": [
+      {
+        "name": "Cortex Xpanse",
+        "min_required": 1,
+        "packs": [
+          {
+            "name": "CortexXpanse",
+            "display_name": "Cortex Xpanse"
+          }
+        ]
+      },
+      {
+        "name": "Cloud Security",
+        "min_required": 0,
```

Comment:
We might want to require these since they playbook will only get remediation guidance without them, but up to you on how you think your current customers would perceive this.

Reply:
We want to make it as actionable as possible with as much ootb content as possible (at first). If we make them required, it might dramatically limit our scope of adoption for this content.... Personally i think we need to add in the incident risk scores, cves and the information from the alert to assess criticality of an incident. For example, we have xdr endpoint protection alert, but come to find out its from a Business critical xpanse tag with port 22 open that was a high risk asset with cve whatever?

<img width="1153" alt="image" src="https://github.com/demisto/content/assets/11729749/97ea225e-285f-4485-bacd-e8150ae3ca67">